src/components/teaser/teaser.scss

```diff
@@ -57,17 +63,17 @@
   flex-flow: var(--sbb-teaser-flex-direction) nowrap;
   align-items: var(--sbb-teaser-align-items);
   gap: var(--sbb-teaser-gap);
-  height: var(--sbb-teaser-container-height, auto);
+  height: auto;
 }
 
 ::slotted([slot='image']) {
   will-change: transform;
   display: block;
   object-fit: cover;
```

Comment:
These are pre-redesign declarations, actually I think that `object-fit` is now unnecessary, since the image is already defined by width and aspect ratio
Instead the `display: block` works with the border-radius of the `.sbb-teaser__image-wrapper` element